mentors/javascript/z.json

```diff
@@ -1 +1,10 @@
-[]
+[
+  {
+    "github_username": "zeckdude",
+    "name": "Chris Seckler",
+    "link_text": "https://github.com/zeckdude",
```

Comment:
```suggestion
    "link_text": null,
```

This uses the `link_url` as label for the link automatically and will avoid forgetting updating the label when you need to change the URL later on.

Reply:
Ok sounds good. Btw, how did you do the commit suggestion in Github? Looks cool!

![Screen Shot 2019-06-10 at 10 13 37 AM](https://user-images.githubusercontent.com/947856/59212904-71959980-8b68-11e9-8d2d-9c11c9291d90.png)